### ISSUE TITLE
Correct issue names to be compatible with file path (on windows)

### ISSUE THREAD

**Issue by mahoromax** [benr77/obsidian-gitlab-issues#8]
When pulling my issues, I noticed these errors:

```
4plugin:obsidian-gitlab-issues:5770 File name cannot contain any of the following characters: * " \ / < > : | ?
plugin:obsidian-gitlab-issues:5770 ENOENT: no such file or directory, open 'C:\git\Obsidian\Work\Gitlab Issues\CI-CD package installation location \ options.md'
plugin:obsidian-gitlab-issues:5770 ENOENT: no such file or directory, open 'C:\git\Obsidian\Work\Gitlab Issues\Collect coverage \ reports \ visualizations for additional jobs.md'
plugin:obsidian-gitlab-issues:5770 ENOENT: no such file or directory, open 'C:\git\Obsidian\Work\Gitlab Issues\Add ID to figure \ file metadata.md'
plugin:obsidian-gitlab-issues:5770 ENOENT: no such file or directory, open 'C:\git\Obsidian\Work\Gitlab Issues\Create option to only add ID to metadata and do NOT on figure \ picture.md'
```

My first idea is to just run regexp over the issue names and replace all of the `* " \ / < > : | ?` with a `-` or `_`  in the filename - but I have no idea about javascript.

**Comment by benr77** [comment]
This is a valid bug and needs to be looked at for sure.

**Comment by benr77** [comment]
Fixed in `0.1.8`
